tests/InputFiles/Elasticity/BentBeam.yaml

```diff
@@ -66,3 +66,5 @@ EventsAndTriggers:
           - Displacement
           - PotentialEnergyDensity
         InterpolateToMesh: None
+        CoordinatesFloatingPointType: Double
+        FloatingPointTypes: [Double, Double]
```

Comment:
I'd suggest the simpler syntax `[Double]` since they're all set to the same type

The file `PlaneWave1DObserveExample.yaml` does a good job of showing the general syntax, so I don't think we need to everywhere.